pkg/watch/registrar.go

```diff
@@ -1,9 +1,26 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package watch
 
 import (
 	"fmt"
 	"sync"
 
+	"sigs.k8s.io/controller-runtime/pkg/event"
```

Comment:
nit: consolidate import groups

Reply:
I've been keeping separation between standard library and external imports. Is that acceptable or do you prefer a flat list mixing both?

Reply:
That separation is fine, there are some places where there are more than 2 import blocks though.

Reply:
Sorry about that, these are artifacts of GoLand / goimports introducing breaks and me not paying much attention to the collapsed blocks. I've fixed where you called them out.

Reply:
ACk, thanks.  I hear you WRT the goimports thing. I get the same thing and try to stay on it, but it's definitely not 100% success rate on my end either.